### ISSUE TITLE
Stl vs Step files

### ISSUE THREAD

**Issue by GlauTechCo** [SoftFever/OrcaSlicer#3927]
### OrcaSlicer Version

1.9.0

### OS version

Macos Sonoma 14.0

### Additional system information

Macbook arm m1 2020

### Printer

Elegoo Neptune 4 Pro

### How to reproduce

Is it normal that there is such a difference between step and stl files?
![flow](https://github.com/SoftFever/OrcaSlicer/assets/33813227/1cf3a16e-9b53-4f40-8e9b-c645bbd617a0)
![3d](https://github.com/SoftFever/OrcaSlicer/assets/33813227/9a0f3d98-65c1-49cd-b19e-fe54a5d97526)

### Actual results

None

### Expected results

None

### Project file & Debug log uploads

[CaseV2.3mf.zip](https://github.com/SoftFever/OrcaSlicer/files/14131084/CaseV2.3mf.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by Eldenroot** [comment]
yes, .stl is much simpler... thats why I prefer .step files (also for exporting models from Fusion etc).

**Comment by Vorlent** [comment]
> > yes, .stl is much simpler... thats why I prefer .step files (also for exporting models from Fusion etc).
> 
> Bambu Studio / Orca are just taking your step file and converting it into a faceted / STL file, but with no control over the quality. I get FAR superior quality exporting STLs from my CAD program than with importing a STEP into BS/Orca.
> 
> I reported this here: [bambulab/BambuStudio#3437](https://github.com/bambulab/BambuStudio/issues/3437)

In this issue, the STL quality is objectively worse compared to STEP.

STL:
![SlicerArtefacts](https://github.com/user-attachments/assets/02801573-e4c7-4cc7-8258-8498cb0dccf3)

STEP:
![SlicerArtefacts2](https://github.com/user-attachments/assets/7f6437f2-9063-43ab-aa0e-d886f54df180)

The first picture has the same problems in the STL preview.

**Comment by CCS86** [comment]
That has to do with your STL export settings.

**Comment by GlauTechCo** [comment]
> That has to do with your STL export settings.

I use onshape as a drawing program. I choose the highest quality offered as stl output quality.
<img width="633" alt="Ekran Resmi 2024-09-19 00 29 23" src="https://github.com/user-attachments/assets/3b720df3-dffc-485d-9792-220d8c78acf7">

**Comment by CCS86** [comment]
> I use onshape as a drawing program. I choose the highest quality offered as stl output quality. >


Onshape's STL translation is still causing that issue.

**Comment by dewi-ny-je** [comment]
> > I use onshape as a drawing program. I choose the highest quality offered as stl output quality. >
> 
> Onshape's STL translation is still causing that issue.

It would be good if at least Orca provided some information about the meshing performed internally, so I can at least know which settings in my CAD software will result in a better model.

**Comment by silvernull** [comment]
It would be useful if you could edit the quality settings,

the surface, normal deviation, max length of edge in mm, max aspect ratio of triangles (the controls fusion 360 gives)

**Comment by movinimage** [comment]
How come we can’t access the meshing settings that BS offers on STEP import?
![IMG_0753](https://github.com/user-attachments/assets/9681d39b-48a3-4dbc-b09f-6c282f5fc0eb)

**Comment by movinimage** [comment]
BS added this dialog in the 1.10 public beta on Oct 15.

**Comment by silvernull** [comment]
> How come we can’t access the meshing settings that BS offers on STEP import?

Really hoping we get this in 2.3.0 so I never have to use 20+MB STL files ever again.

**Comment by 1951FDG** [comment]
Me as well!! Will this be included in 2.3.0?
